automation_tools/__init__.py

```diff
@@ -2130,12 +2163,20 @@ def satellite6_upgrade(admin_password=None):
     run('rm -rf /etc/yum.repos.d/rhel-{optional,released}.repo')
     print('Wait till Packages update ... ')
     update_packages(quiet=True)
+    # Rebooting the system to see possible errors
+    reboot(env.get('satellite_host'), 120)
     # Setting Satellite61 Repos
     major_ver = distro_info()[1]
     base_url = os.environ.get('BASE_URL')
     if base_url is None:
         enable_repos('rhel-{0}-server-satellite-6.1-rpms'.format(major_ver))
-    disable_repos('rhel-{0}-server-satellite-6.0-rpms'.format(major_ver))
+    if from_version == '6.1':
```

Comment:
Assuming that the expected version was already validated you can turn the following 4 lines into:

```
disable_repos('rhel-{0}-server-satellite-{1}-rpms'.format(major_ver, from_version))
```